axelor-sale/src/main/java/com/axelor/apps/sale/rest/dto/SaleOrderPostRequest.java

```diff
@@ -77,4 +88,11 @@ public Currency fetchCurrency() {
     }
     return ObjectFinder.find(Currency.class, currencyId, ObjectFinder.NO_VERSION);
   }
+
+  public List<SaleOrderLinePostRequest> fetchSaleOrderLinePostRequests() {
+    if (saleOrderLinePostRequests == null || saleOrderLinePostRequests.isEmpty()) {
```

Comment:
Use CollectionUtils method. And the method should be named getSaleOrderLinePostRequests since we are not fetching existing data from the database.